internal/cli/capact/components.go

```diff
@@ -648,12 +645,23 @@ func createObject(configuration *action.Configuration, content []byte) error {
 		return errors.Wrap(err, "while validating the object")
 	}
 
-	if _, err := configuration.KubeClient.Create(res); err != nil {
-		// If the error is CRD already exists, return.
-		if apierrors.IsAlreadyExists(err) {
-			return nil
-		}
-		return errors.Wrapf(err, "while creating the object")
+	// 5 exponential retries: ~2µs ~12ms ~70ms ~325ms ~1.63s
+	retryBackoff := retry.DefaultBackoff
+	retryBackoff.Steps = 5
+	err = retry.OnError(retryBackoff, differentThanAlreadyExistErr, func() error {
```

Comment:
🤔 Usually we use `github.com/avast/retry-go` in such scenario. There are two places where we use `k8s.io/client-go/util/retry` - here and in `k8s.go` file, also for installation.

We can keep it as it is, but I think long-term it would be great to have consistency and use just a single package for a single thing. Same what we have with multierrors.